Iss hipcms 77 achievements

ruchirnaphade opened the pull request:
Create, edit and delete achievement with image
choose exhibit and choose route is not working as we need data regarding the same from server.

Kindly do not review old PR for achievements i.e. iss-hipangular-77-create-achievement-dialog

Commit:
again added edit achievement dialog box (first one lost due to some issues),added create,delete method for achievements

Commit:
made changes in edit achievement box,created edit method

Commit:
Resolving conflicts to sync with hipangular-77 branch

Commit:
created achievement dialog box,added empty achievement methods for both achievement types

Commit:
Merge branch 'iss-hipangular-77' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-hipangular-77-create-achievement-dialog

Commit:
trying to add empty method sample() for getting type

Commit:
achievement method created and call successfully both the APIs

Commit:
implemented delete method to delete achievements by clicking delete icon

Commit:
created and implemented edit achievement method successfully and made changes in edit achievement page.Furthermore trying to implement image upload and still working on it

Commit:
added image upload dialog in edit achievement page and formated my code

Commit:
trying to add properties for achievement type

Commit:
Merge branch 'iss-hipangular-77' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-hipangular-77-create-achievement-dialog

Commit:
changed get all users service from cms-api to userstore

Commit:
changed update user API call from cmsapi to userstore

Commit:
successfully implemented upload image API for admin user from userstore

Commit:
Adding temporary changes for upload image

Commit:
resolving conflicts and merging with  iss-hipangular-77

Commit:
implemented achievement creation with file upload but getting unauthorized error

Commit:
implemented create achievement with image upload and edit achievement with image upload successfully

Commit:
trying to implement edit achievement method without image

Commit:
tried to implement update image in edit achievement

Commit:
create and edit achievement implemented successfully

Commit:
fixed build error for my pull request

Commit:
fixed type annotation error

Commit:
fixed type annotation error

Commit:
resolved conflicts

Commit:
resolved conflicts again

pschick committed:
Merge branch 'develop' into iss-hipangular-77-create-achievement-dialog

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-793

Commit:
merge develop into achievements branch and created new branch for achievements

Commit:
created new PR for achievements

Commit:
fixed tlint error

Commit:
Merge branch 'iss-HIPCMS-793' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-793

Commit:
deleted exhibits select option from create achievement

Commit:
removed package-lock file

Commit:
Merge branch 'iss-HIPCMS-793' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-793

Commit:
changed hip config file

Commit:
fixed errors

Commit:
fixed tslint error

Commit:
resolved conflicts

objectliteral committed:
Merge branch 'develop' into iss-HIPCMS-77-achievements

Commit:
removed select bar from exhibit visited achivement

pschick committed:
Merge branch 'develop' into iss-HIPCMS-77-achievements

pschick reviewed (dismissed):
TODO:
- I can't select an exhibit in the dropdown when trying to create an "Exhibits visited" achievement
- The "With status" filter doesn't work when "In Review" is selected
- Restrict the file upload to the supported file types (e.g. png is not a supported file type)

Commit:
removed select box and png type

Commit:
removed png type

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-793

Commit:
Merge branch 'iss-HIPCMS-77-achievements' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-77-achievements

Commit:
removed error

ruchirnaphade commented:
- Removed select box from exhibit visited
- with status is not working because there is something wrong with backend. (asked michel and confirmed)
- Removed png type

Commit:
Merge branch 'iss-HIPCMS-793' of https://github.com/HiP-App/HiP-CmsAngularApp into iss-HIPCMS-793

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into develop

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into develop

Commit:
Merge branch 'iss-hipcms-969' of https://github.com/HiP-App/HiP-CmsAngularApp into develop

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into develop

Commit:
changes according to develop

Commit:
change according to new develop

Commit:
conflicts resolved

Commit:
Merge branch 'develop' of https://github.com/HiP-App/HiP-CmsAngularApp into develop

Commit:
resolved conflicts

Commit:
resolved conflicts

Commit:
resolved conflicts

Commit:
resolved conflicts

pschick committed:
Merge branch 'develop' into iss-HIPCMS-77-achievements

rajatkhanna1994 committed:
Merge branch 'develop' into iss-HIPCMS-77-achievements

rajatkhanna1994 committed:
Merge branch 'develop' into iss-HIPCMS-77-achievements

rajatkhanna1994 reviewed (changes requested):
When creating a new achievement, I'm not able to view it on the Achievements page. Only after refreshing it, I'm able to see the newly created achievement.
Could you look into it?
